Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -549,7 +550,10 @@ class TabTrayController: UIViewController, UITabBarDelegate, UICollectionViewDel
             } else {
                 cell.favicon.image = UIImage(named: "defaultFavicon")
             }
-            cell.background.image = tab.screenshot
+
+            if let img = tab.screenshot {
+                cell.background.image = img
+            }
```

Comment:
Nit: I think this is an unnecessary change since setting `cell.background.image` to `nil` is the same as not setting it at all.